Starcraft2Client.py

```diff
@@ -897,17 +928,52 @@ def check_game_install_path() -> bool:
 
 
 def check_mod_install() -> bool:
```

Comment:
I'd suggest a rename on functions like this to provide greater clarity when reading just the name. `check_mod_install` is somewhat nebulous but something like `is_mod_installed_correctly` is more clear and due to the verb `is` it is more clear that it returns a `bool`.